### PR TITLE
Improvements for BET-18

### PR DESCRIPTION
**Pull request by noahwc**
## Description of the change

> Description here

## Type of change

- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation or Development tools (readme, specs, tests, code formatting)

## Links

- Jira issue number: (PUT IT HERE)
- Process.st launch checklist: (PUT IT HERE)

### Development

- [ ] [Prettier](https://www.npmjs.com/package/prettier) was run (if applicable)
- [ ] The behaviour changes in the pull request are covered by specs
- [ ] All tests related to the changed code pass in development

### Paperwork

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has a Jira number
- [ ] This pull request has a Process.st launch checklist

### Code review

- [ ] Changes have been reviewed by at least one other engineer
- [ ] Security impacts of this change have been considered

### CODE REVIEW

File: packages/visual-dev/src/components/RadioCard/RadioCard.tsx [hunk 1]
```diff
@@ -67,45 +73,54 @@ const RadioGridDiv = styled.div`
...
+              ""
+            )}
+            {description ? (
+              <div style={{ color: "var(--sq-text-subdued)" }}>
+                {description}
+              </div>
+            ) : (
+              ""
+            )}
+          </RadioTextDiv>
```
Comment (tdgao) [inline packages/visual-dev/src/components/RadioCard/RadioCard.tsx:118]: Can be cleaned up by switching ? to &&  
```suggestion
          <RadioTextDiv>
            {title && (
              <div style={{ fontWeight: "bold", marginBottom: 4 }}>{title}</div>
            ) }
            {description && (
              <div style={{ color: "var(--sq-text-subdued)" }}>
                {description}
              </div>
            )}
          </RadioTextDiv>
```

**Review by zachharrison — APPROVED** [review]
Looks good, just need to make sure we accept the radio card changes from bet-85.

File: packages/visual-dev/src/components/RadioCard/RadioCard.tsx (hunk 1 above)
Comment (zachharrison) [reply packages/visual-dev/src/components/RadioCard/RadioCard.tsx:118]: Fixed in the bet-85 branch

**Comment by noahwc** [comment]
Closed in favor of #332

**Pull request closed without merging**